### PR TITLE
chore(cas): Renames to use stream terminology throughout code

### PR DESCRIPTION
**Pull request by stbrody**
One thing that still needs to be decided is what we're going to do about the [docid field](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/886fd416f2b0c8d9f3bf011a58fc894d35be297a/src/merkle/merkle-objects.ts#L76) in the bloom filter in the merkle tree metadata. If we don't change it now, then we're stuck using `docid` literally forever, confusing the story when working with indexer partners and adding complication and confusion for the rest of time.  If we *do* change it now, then any streams created in the first week of ELP won't get indexed - at least until they are updated again and get a new AnchorCommit.  Personally I kind of think we should just make the cut now and tell our ELP cohort 1 to do a trivial update to any streams they want to get indexed.  The likelihood of those streams not getting indexed actually ever being an issue seems really low, not worth adding complexity to the system for the rest of all time over.

We could even make the change now and instruct indexing partners in the future to check for streamId first and fall back to checking for docId

@oed @michaelsena

### CODE REVIEW

**Comment by stbrody** [comment]
FYI @v-stickykeys

File: src/controllers/request-controller.ts [hunk 1]
```diff
@@ -63,17 +63,20 @@ export default class RequestController {
     try {
       logger.debug(`Create request ${JSON.stringify(req.body)}`);
 
-      const { cid, docId } = req.body;
+      const cid = req.body.cid;
+      // TODO docId check for backwards compat with old Ceramic nodes. Remove once all infra nodes
+      // updated to new Ceramic version
```
Comment (oed) [inline src/controllers/request-controller.ts:68]: We should probably keep this around for longer. Ppl are using their own nodes on clay. Not only tied to our infra.
```suggestion
      // TODO docId check for backwards compat with old Ceramic nodes. Remove once confident that
      // most nodes in the network updated to new Ceramic version
```

Comment (oed) [reply src/controllers/request-controller.ts:68]: Please also indicate which version here for future reference.

File: src/merkle/merkle-objects.ts [hunk 2]
```diff
@@ -72,20 +72,20 @@ export class BloomMetadata implements MetadataFunction<Candidate, TreeMetadata>
   generateMetadata(leaves: Array<Node<Candidate>>): TreeMetadata {
     const bloomFilterEntries = new Set<string>()
     for (const node of leaves) {
-      const doc = node.data.document
-      bloomFilterEntries.add(`docid-${doc.id.baseID.toString()}`)
-      if (doc.metadata.schema) {
-        bloomFilterEntries.add(`schema-${doc.metadata.schema.toString()}`)
+      const stream = node.data.stream
+      bloomFilterEntries.add(`docid-${stream.id.baseID.toString()}`)
```
Comment (oed) [inline src/merkle/merkle-objects.ts:76]:
```suggestion
      bloomFilterEntries.add(`streamid-${stream.id.baseID.toString()}`)
```

**Review by oed — CHANGES_REQUESTED** [review]

File: src/controllers/request-controller.ts (hunk 1 above)
Comment (stbrody) [reply src/controllers/request-controller.ts:68]: I won't know the version number to use before merging this.  I need to get this merged, released, and deployed on our infra, then I can make the change to ceramic to use the new format and get that released.

I'll include a reference to the ceramic github issue that will handle the support, that seems like the best we can do

Comment (oed) [reply src/controllers/request-controller.ts]: I guess you could just include the js-ceramic version we are currently at? Or better, a reference to the issue for fixing it!

**Review by oed — APPROVED** [review]

File: src/controllers/request-controller.ts (hunk 1 above)
Comment (oed) [reply src/controllers/request-controller.ts]: Ah, that's what you did 👍

**Pull request merged**